CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 
 <!-- changelog start -->
 
+## [0.10.0 (2024-01-24)]
+
+
+### New Features
+
+- Introduced an unused expression checker. It's designed to detect and report unused expressions,
+  helping users to clean up and optimize their VRL scripts. Note that this checker will not catch everything, 
+  but it does aim to eliminate false positives. For example, shadowed variables are not reported as unused. 
```

Comment:
strange that this didn't render a PR link hmm

Reply:
Could be that it has multiple newlines or it is too long?